service.ifttt/addon.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon
+	id="service.ifttt"
+	name="IFTTT Service"
+	version="1.0.0"
+	provider-name="plucka"
+	>
+	<requires>
+		<import addon="xbmc.python" version="2.26.0"/>
+	</requires>
+	<extension point="xbmc.service" library="default.py" start="login"/>
+	<extension point="xbmc.python.script" library="default.py">
+		<provides>executable</provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en">IFTTT Service</summary>
+		<description lang="en">Calls IFTTT based on events</description>
+		<disclaimer lang="en">Open Source</disclaimer>
```

Comment:
en_GB (also please improve the disclaimer text, "open source" doesn't tell much :)